### DISCUSSION TITLE
How can I change the port that ComfyUI uses?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [Comfy-Org/ComfyUI#986] [Q&A] [answered]
I ran it, stopped it, and restarted it and got an error saying the port is in use. I'd like to know how to set a custom port still though.

**Comment by n00mkrad** [comment, marked as answer]
`--port`

**Reply by YousufSSyed** [reply]
I had tried that last week, idk if it will work if I try it again for awhile, but I'll mark it as solved for now.

**Reply by Hybris95** [reply]
Tested, works like a charm, allows me to circumvent a PermissionError
Is there a way to put this as default in the comfy.settings.json ?
